PontoFacil/PontoFacil/ViewModels/PlanningPageViewModel.cs

```diff
@@ -41,5 +41,20 @@ private void ActionSave()
             _planningService.Save(Planning);
         }
         #endregion
+
+        public void TextBox_TextChanging(TextBox sender, TextBoxTextChangingEventArgs args)
+        {
+            if (!IsANumber(sender.Text))
+            {
+                int pos = sender.SelectionStart - 1;
+                sender.Text = sender.Text.Remove(pos, 1);
+                sender.SelectionStart = pos + 1;
```

Comment:
I didn't get it what this block of code should do, maybe if you refactor It will become clearer.